### DISCUSSION TITLE
Problem with Multiple CF on Crazyswarm hover_swarm

### DISCUSSION THREAD

**Discussion by tbaranoski** [USC-ACTLab/crazyswarm#451] [Q&A]
I have two crazyflies I am trying to hook up to the crazyswarm hover_swarm. My end goal is too use ROS to share CF#1 sensory information with CF#2. I do not intend to fly CF#2. I am not using any tracking systems. I am running into errors when trying to use two CF with hover_swarm. Any help would be appreciated.

CF `#1` has a flow deck. Address: 0xE7E7E7E701 0/100/2M
CF `#2` does not have a flow deck. Address: 0xE7E7E7E701 0/110/2M

My allCrazyflies.yaml file look like:

`crazyflies:
  - id: 1 `#1`
    channel: 100 `#100`
    initialPosition: [1.5, 1.5, 0.0]
    type: medium
  - id: 2
    channel: 110
    initialPosition: [-1.0, -0.5, 0.0]
    type: medium
`

I used chooser.py to select both check boxes for both CF#1 and CF#2. crazylies.yaml reflect these changes. However I get errors while running hover_swarm. The terminal output is shown below. To me, it looks like CF#1 connects to ROS server fine but CF#2 does not. Both CF#1 and CF#2 works on hover_swarm when they are the only CF.

`roslaunch crazyswarm hover_swarm.launch          
... logging to /home/bitcraze/.ros/log/dbf8c564-f6d1-11eb-a8eb-5be5295ad037/roslaunch-ubuntu-3053.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://ubuntu:41359/

SUMMARY
========

PARAMETERS
 * /crazyflieTypes/CF21SingleMarker/batteryVoltageWarning: 3.8
 * /crazyflieTypes/CF21SingleMarker/batteryVoltateCritical: 3.7
 * /crazyflieTypes/CF21SingleMarker/bigQuad: False
 * /crazyflieTypes/CF21SingleMarker/dynamicsConfiguration: 0
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/i_range_m_z: 1500
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/i_range_xy: 2.0
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/i_range_z: 0.4
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kR_xy: 70000
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kR_z: 60000
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kd_omega_rp: 200
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kd_xy: 0.2
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kd_z: 0.4
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/ki_m_z: 500
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/ki_xy: 0.05
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/ki_z: 0.05
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kp_xy: 0.4
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kp_z: 1.25
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kw_xy: 20000
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/kw_z: 12000
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/mass: 0.032
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/ctrlMel/massThrust: 132000
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/kalman/mNGyro_rollpitch: 0.1
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/kalman/mNGyro_yaw: 0.1
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/kalman/pNAcc_xy: 1.0
 * /crazyflieTypes/CF21SingleMarker/firmwareParams/kalman/pNAcc_z: 2.0
 * /crazyflieTypes/CF21SingleMarker/markerConfiguration: 3
 * /crazyflieTypes/default/batteryVoltageWarning: 3.8
 * /crazyflieTypes/default/batteryVoltateCritical: 3.7
 * /crazyflieTypes/default/bigQuad: False
 * /crazyflieTypes/default/dynamicsConfiguration: 0
 * /crazyflieTypes/default/firmwareParams/ctrlMel/i_range_m_z: 1500
 * /crazyflieTypes/default/firmwareParams/ctrlMel/i_range_xy: 2.0
 * /crazyflieTypes/default/firmwareParams/ctrlMel/i_range_z: 0.4
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kR_xy: 70000
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kR_z: 60000
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kd_omega_rp: 200
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kd_xy: 0.2
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kd_z: 0.4
 * /crazyflieTypes/default/firmwareParams/ctrlMel/ki_m_z: 500
 * /crazyflieTypes/default/firmwareParams/ctrlMel/ki_xy: 0.05
 * /crazyflieTypes/default/firmwareParams/ctrlMel/ki_z: 0.05
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kp_xy: 0.4
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kp_z: 1.25
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kw_xy: 20000
 * /crazyflieTypes/default/firmwareParams/ctrlMel/kw_z: 12000
 * /crazyflieTypes/default/firmwareParams/ctrlMel/mass: 0.032
 * /crazyflieTypes/default/firmwareParams/ctrlMel/massThrust: 132000
 * /crazyflieTypes/default/markerConfiguration: 0
 * /crazyflieTypes/defaultSingleMarker/batteryVoltageWarning: 3.8
 * /crazyflieTypes/defaultSingleMarker/batteryVoltateCritical: 3.7
 * /crazyflieTypes/defaultSingleMarker/bigQuad: False
 * /crazyflieTypes/defaultSingleMarker/dynamicsConfiguration: 0
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/i_range_m_z: 1500
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/i_range_xy: 2.0
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/i_range_z: 0.4
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kR_xy: 70000
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kR_z: 60000
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kd_omega_rp: 200
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kd_xy: 0.2
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kd_z: 0.4
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/ki_m_z: 500
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/ki_xy: 0.05
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/ki_z: 0.05
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kp_xy: 0.4
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kp_z: 1.25
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kw_xy: 20000
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/kw_z: 12000
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/mass: 0.032
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/ctrlMel/massThrust: 132000
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/kalman/mNGyro_rollpitch: 0.2
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/kalman/mNGyro_yaw: 0.1
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/kalman/pNAcc_xy: 4.0
 * /crazyflieTypes/defaultSingleMarker/firmwareParams/kalman/pNAcc_z: 8.0
 * /crazyflieTypes/defaultSingleMarker/markerConfiguration: 3
 * /crazyflieTypes/large/batteryVoltageWarning: 11.4
 * /crazyflieTypes/large/batteryVoltateCritical: 11.1
 * /crazyflieTypes/large/bigQuad: True
 * /crazyflieTypes/large/dynamicsConfiguration: 0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/i_range_m_z: 1500
 * /crazyflieTypes/large/firmwareParams/ctrlMel/i_range_xy: 2.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/i_range_z: 4.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kR_xy: 30000
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kR_z: 40000
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kd_omega_rp: 100
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kd_xy: 3.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kd_z: 3.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/ki_m_z: 500
 * /crazyflieTypes/large/firmwareParams/ctrlMel/ki_xy: 1.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/ki_z: 2.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kp_xy: 8.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kp_z: 8.0
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kw_xy: 9000
 * /crazyflieTypes/large/firmwareParams/ctrlMel/kw_z: 10000
 * /crazyflieTypes/large/firmwareParams/ctrlMel/mass: 0.38
 * /crazyflieTypes/large/firmwareParams/ctrlMel/massThrust: 5400
 * /crazyflieTypes/large/markerConfiguration: 2
 * /crazyflieTypes/medium/batteryVoltageWarning: 7.6
 * /crazyflieTypes/medium/batteryVoltateCritical: 7.4
 * /crazyflieTypes/medium/bigQuad: True
 * /crazyflieTypes/medium/dynamicsConfiguration: 0
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/i_range_m_z: 1500
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/i_range_xy: 2.0
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/i_range_z: 2.0
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kR_xy: 50000
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kR_z: 60000
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kd_omega_rp: 200
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kd_xy: 1.0
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kd_z: 1.0
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/ki_m_z: 500
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/ki_xy: 0.1
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/ki_z: 0.5
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kp_xy: 2.0
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kp_z: 3
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kw_xy: 16000
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/kw_z: 12000
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/mass: 0.11
 * /crazyflieTypes/medium/firmwareParams/ctrlMel/massThrust: 23000
 * /crazyflieTypes/medium/markerConfiguration: 1
 * /crazyflies: [{'channel': 100,...
 * /crazyswarm_server/enable_logging: True
 * /crazyswarm_server/enable_parameters: True
 * /crazyswarm_server/firmwareParams/commander/enHighLevel: 1
 * /crazyswarm_server/firmwareParams/kalman/resetEstimation: 1
 * /crazyswarm_server/firmwareParams/locSrv/extPosStdDev: 1e-3
 * /crazyswarm_server/firmwareParams/locSrv/extQuatStdDev: 0.05
 * /crazyswarm_server/firmwareParams/ring/effect: 16
 * /crazyswarm_server/firmwareParams/ring/headlightEnable: 0
 * /crazyswarm_server/firmwareParams/ring/solidBlue: 255
 * /crazyswarm_server/firmwareParams/ring/solidGreen: 0
 * /crazyswarm_server/firmwareParams/ring/solidRed: 0
 * /crazyswarm_server/firmwareParams/stabilizer/controller: 2
 * /crazyswarm_server/firmwareParams/stabilizer/estimator: 2
 * /crazyswarm_server/force_no_cache: False
 * /crazyswarm_server/genericLogTopicFrequencies: [10]
 * /crazyswarm_server/genericLogTopic_log1_Variables: ['stateEstimate.x...
 * /crazyswarm_server/genericLogTopics: ['log1']
 * /crazyswarm_server/motion_capture_type: none
 * /crazyswarm_server/object_tracking_type: libobjecttracker
 * /crazyswarm_server/print_latency: False
 * /crazyswarm_server/save_point_clouds: ~/pointCloud.ot
 * /crazyswarm_server/send_position_only: False
 * /crazyswarm_server/vicon_host_name: vicon
 * /crazyswarm_server/world_frame: /world
 * /crazyswarm_server/write_csvs: False
 * /crazyswarm_teleop/csv_file: /home/bitcraze/cr...
 * /crazyswarm_teleop/timescale: 0.8
 * /dynamicsConfigurations/0/maxFitnessScore: 0.001
 * /dynamicsConfigurations/0/maxPitch: 1.4
 * /dynamicsConfigurations/0/maxPitchRate: 20.0
 * /dynamicsConfigurations/0/maxRoll: 1.4
 * /dynamicsConfigurations/0/maxRollRate: 20.0
 * /dynamicsConfigurations/0/maxXVelocity: 2.0
 * /dynamicsConfigurations/0/maxYVelocity: 2.0
 * /dynamicsConfigurations/0/maxYawRate: 10.0
 * /dynamicsConfigurations/0/maxZVelocity: 3.0
 * /joy/dev: /dev/input/js2
 * /markerConfigurations/0/numPoints: 4
 * /markerConfigurations/0/offset: [0.0, -0.01, -0.04]
 * /markerConfigurations/0/points/0: [0.0177184, 0.013...
 * /markerConfigurations/0/points/1: [-0.0262914, 0.05...
 * /markerConfigurations/0/points/2: [-0.0328889, -0.0...
 * /markerConfigurations/0/points/3: [0.0431307, -0.03...
 * /markerConfigurations/1/numPoints: 4
 * /markerConfigurations/1/offset: [0.0, 0.0, -0.03]
 * /markerConfigurations/1/points/0: [-0.00896228, -0....
 * /markerConfigurations/1/points/1: [-0.0156318, 0.09...
 * /markerConfigurations/1/points/2: [0.0461693, -0.08...
 * /markerConfigurations/1/points/3: [-0.0789959, -0.0...
 * /markerConfigurations/2/numPoints: 4
 * /markerConfigurations/2/offset: [0.0, 0.0, -0.06]
 * /markerConfigurations/2/points/0: [0.0558163, -0.00...
 * /markerConfigurations/2/points/1: [-0.0113941, 0.00...
 * /markerConfigurations/2/points/2: [-0.0306277, 0.05...
 * /markerConfigurations/2/points/3: [0.0535816, -0.04...
 * /markerConfigurations/3/numPoints: 1
 * /markerConfigurations/3/offset: [0.0, -0.01, -0.04]
 * /markerConfigurations/3/points/0: [0.0177184, 0.013...
 * /numDynamicsConfigurations: 1
 * /numMarkerConfigurations: 4
 * /rosdistro: noetic
 * /rosversion: 1.15.11

NODES
  /
    crazyswarm_server (crazyswarm/crazyswarm_server)
    crazyswarm_teleop (crazyswarm/crazyswarm_teleop)
    joy (joy/joy_node)
    rviz (rviz/rviz)

auto-starting new master
process[master]: started with pid [3080]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to dbf8c564-f6d1-11eb-a8eb-5be5295ad037
process[rosout-1]: started with pid [3097]
started core service [/rosout]
process[crazyswarm_server-2]: started with pid [3104]
process[joy-3]: started with pid [3105]
process[crazyswarm_teleop-4]: started with pid [3106]
[ERROR] [1628266678.087697095]: Couldn't open joystick /dev/input/js2. Will retry every second.
process[rviz-5]: started with pid [3107]
[ INFO] [1628266678.119136155]: Wait for services...
[ INFO] [1628266678.119994068]: waitForService: Service [/emergency] has not been advertised, waiting...
[ INFO] [1628266678.544976960]: waitForService: Service [/emergency] is now available.
[ INFO] [1628266678.546689580]: Manager ready.
ch: 2
[ INFO] [1628266678.677893866]: Adding CF: cf1 (radio://0/100/2M/E7E7E7E701, cf1)...
[ INFO] [1628266678.723732460]: CF ctor: 0.045039 s
[ INFO] [1628266678.760705388]: [cf1] SYS: ----------------------------
[ INFO] [1628266678.780288223]: [cf1] SYS: Crazyflie 2.1 is up and running!
[ INFO] [1628266678.802749335]: [cf1] SYS: Build 11:0864ef92245a (2021.03 +11) CLEAN
[ INFO] [1628266678.837828947]: [cf1] SYS: I am 0x20383347344D50080030004D and I have 1024KB of flash!
[ INFO] [1628266678.849535604]: [cf1] CFGBLK: v1, verification [OK]
[ INFO] [1628266678.861204720]: [cf1] DECK_CORE: 2 deck(s) found
[ INFO] [1628266678.883525080]: [cf1] DECK_CORE: Calling INIT on driver bcFlow2 for deck 0
[ INFO] [1628266678.894505518]: [cf1] ZR2: Z-down sensor [OK]
[ INFO] [1628266678.917370859]: [cf1] PMW: Motion chip id: 0x49:0xB6
[ INFO] [1628266678.939536450]: [cf1] DECK_CORE: Calling INIT on driver bcMultiranger for deck 1
[ INFO] [1628266678.959898134]: [cf1] IMU: BMI088: Using I2C interface.
[ INFO] [1628266678.982338164]: [cf1] IMU: BMI088 Gyro connection [OK].
[ INFO] [1628266679.004211997]: [cf1] IMU: BMI088 Accel connection [OK]
[ INFO] [1628266679.025659046]: [cf1] IMU: BMP388 I2C connection [OK]
[ INFO] [1628266679.048826512]: [cf1] ESTIMATOR: Using Kalman (2) estimator
[ INFO] [1628266679.071435184]: [cf1] CONTROLLER: Using PID (1) controller
[ INFO] [1628266679.093989414]: [cf1] MTR-DRV: Using brushed motor driver
[ INFO] [1628266679.104773325]: [cf1] EEPROM: I2C connection [OK].
[ INFO] [1628266679.115435891]: [cf1] STORAGE: Storage check [OK].
[ INFO] [1628266679.135967237]: [cf1] IMU: BMI088 gyro self-test [OK]
[ INFO] [1628266679.147724473]: [cf1] DECK_CORE: Deck 0 test [OK].
[ INFO] [1628266679.158467365]: [cf1] MR: Init front sensor [OK]
[ INFO] [1628266679.170131545]: [cf1] MR: Init back sensor [OK]
[ INFO] [1628266679.179862834]: [cf1] MR: Init up sensor [OK]
[ INFO] [1628266679.189571380]: [cf1] MR: Init left sensor [OK]
[ INFO] [1628266679.201388034]: [cf1] MR: Init right sensor [OK]
[ INFO] [1628266679.213037576]: [cf1] DECK_CORE: Deck 1 test [OK].
[ INFO] [1628266679.238437240]: [cf1] STAB: Wait for sensor calibration...
[ INFO] [1628266679.250245928]: [cf1] SYS: Free heap: 16352 bytes
[ INFO] [1628266679.262853509]: [cf1] STAB: Ready to fly.
[ INFO] [1628266679.278519650]: [cf1] Requesting parameters...
[ INFO] [1628266679.737008621]: Found variables in cache.
[ INFO] [1628266682.566957313]: [cf1] reqParamTOC: 3.843128 s (229 params)
[ INFO] [1628266682.567034474]: [cf1] Requesting logging variables...
[ INFO] [1628266682.601586532]: Found variables in cache.
[ INFO] [1628266682.602777832]: [cf1] reqLogTOC: 0.035872 s
[ INFO] [1628266682.667187008]: [cf1] logBlocks: 3.843128 s
[ INFO] [1628266682.745207394]: Requesting memories...
[ INFO] [1628266682.838331526]: Memories: 6
[ INFO] [1628266683.148241669]: [cf1] Ready. Elapsed: 4.424458 s
[ INFO] [1628266683.148274714]: CF run: 4.424558 s
update commander/enHighLevel to 1
update kalman/resetEstimation to 1
update locSrv/extPosStdDev to 0.001
update locSrv/extQuatStdDev to 0.05
update ring/effect to 16
update ring/headlightEnable to 0
update ring/solidBlue to 255
update ring/solidGreen to 0
update ring/solidRed to 0
update stabilizer/controller to 2
update stabilizer/estimator to 2
update ctrlMel/i_range_m_z to 1500
update ctrlMel/i_range_xy to 2
update ctrlMel/i_range_z to 2
update ctrlMel/kR_xy to 50000
update ctrlMel/kR_z to 60000
update ctrlMel/kd_omega_rp to 200
update ctrlMel/kd_xy to 1
update ctrlMel/kd_z to 1
update ctrlMel/ki_m_z to 500
update ctrlMel/ki_xy to 0.1
update ctrlMel/ki_z to 0.5
update ctrlMel/kp_xy to 2
update ctrlMel/kp_z to 3
update ctrlMel/kw_xy to 16000
update ctrlMel/kw_z to 12000
update ctrlMel/mass to 0.11
update ctrlMel/massThrust to 23000
[ INFO] [1628266683.161499130]: [cf1] Update parameters
[ INFO] [1628266683.829707076]: [cf1] CONTROLLER: Using Mellinger (2) controller
[ INFO] [1628266684.263032549]: Update params: 1.114728 s
terminate called after throwing an instance of 'std::runtime_error'
  what():  No matching USB Device with devid = 1 found! Do you have a Crazyradio dongle connected?
[crazyswarm_server-2] process has died [pid 3104, exit code -6, cmd /home/bitcraze/crazyswarm/ros_ws/devel/lib/crazyswarm/crazyswarm_server __name:=crazyswarm_server __log:=/home/bitcraze/.ros/log/dbf8c564-f6d1-11eb-a8eb-5be5295ad037/crazyswarm_server-2.log].
log file: /home/bitcraze/.ros/log/dbf8c564-f6d1-11eb-a8eb-5be5295ad037/crazyswarm_server-2*.log
^C[rviz-5] killing on exit
`

In addition I am trying to look at the crazyswarm_server-2*.log and it is no where in the directory. Below is a snapshot of the directory (including hidden files). Any reason why the log file is not showing up?

![no_log](https://user-images.githubusercontent.com/84639789/128542598-33deecf8-ea96-4d8a-86cd-0eacf5e03cd1.PNG)

**Comment by jpreiss** [comment]
You are configured to use channels 100 and 110. Are you using two crazyradios?

**Reply by tbaranoski** [reply]
I am using one crazyradio. Is that the problem? Should they both have the same channel and just have a different address instead?

**Reply by jpreiss** [reply]
yes.

**Reply by tbaranoski** [reply]
Thank you!
It is working now.
